src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ReservedKeywordPositive.txt

```diff
@@ -0,0 +1,82 @@
+
+>> {null    :1}
+Errors: Error 1-5: Unexpected characters. The formula contains 'Error' where 'Ident' is expected.|Error 1-5: Unexpected characters. The formula contains 'Error' where 'Colon' is expected.|Error 1-5: Expected colon. We expect a colon (:) at this point in the formula.|Error 9-10: Unexpected characters. The formula contains 'Colon' where 'CurlyClose' is expected.|Error 9-10: Unexpected characters. Characters are used in the formula in an unexpected way.
+
+>> {none    :1}
+Errors: Error 1-5: Unexpected characters. The formula contains 'Error' where 'Ident' is expected.|Error 1-5: Unexpected characters. The formula contains 'Error' where 'Colon' is expected.|Error 1-5: Expected colon. We expect a colon (:) at this point in the formula.|Error 9-10: Unexpected characters. The formula contains 'Colon' where 'CurlyClose' is expected.|Error 9-10: Unexpected characters. Characters are used in the formula in an unexpected way.
+
+>> {nothing :1}
+Errors: Error 1-8: Unexpected characters. The formula contains 'Error' where 'Ident' is expected.|Error 1-8: Unexpected characters. The formula contains 'Error' where 'Colon' is expected.|Error 1-8: Expected colon. We expect a colon (:) at this point in the formula.|Error 9-10: Unexpected characters. The formula contains 'Colon' where 'CurlyClose' is expected.|Error 9-10: Unexpected characters. Characters are used in the formula in an unexpected way.
+
+>> {undefined:1}
+Errors: Error 1-10: Unexpected characters. The formula contains 'Error' where 'Ident' is expected.|Error 1-10: Unexpected characters. The formula contains 'Error' where 'Colon' is expected.|Error 1-10: Expected colon. We expect a colon (:) at this point in the formula.|Error 10-11: Unexpected characters. The formula contains 'Colon' where 'CurlyClose' is expected.|Error 10-11: Unexpected characters. Characters are used in the formula in an unexpected way.
+
+>> {Children:1}
+Errors: Error 1-9: Unexpected characters. The formula contains 'Error' where 'Ident' is expected.|Error 1-9: Unexpected characters. The formula contains 'Error' where 'Colon' is expected.|Error 1-9: Expected colon. We expect a colon (:) at this point in the formula.|Error 9-10: Unexpected characters. The formula contains 'Colon' where 'CurlyClose' is expected.|Error 9-10: Unexpected characters. Characters are used in the formula in an unexpected way.
+
+>> {Siblings:1}
+Errors: Error 1-9: Unexpected characters. The formula contains 'Error' where 'Ident' is expected.|Error 1-9: Unexpected characters. The formula contains 'Error' where 'Colon' is expected.|Error 1-9: Expected colon. We expect a colon (:) at this point in the formula.|Error 9-10: Unexpected characters. The formula contains 'Colon' where 'CurlyClose' is expected.|Error 9-10: Unexpected characters. Characters are used in the formula in an unexpected way.
+
+>> {'OkToUse' :1}
```

Comment:
also test `X Is Y`, should result in the reserved error

Reply:
Added.  Getting odd results and not seeing my error message.  I'll need to dig into this.